src/test/groovy/school/lemon/changerequest/java/introduction/hw1/Task3Specification.groovy

```diff
@@ -16,5 +16,6 @@ class Task3Specification extends Specification {
         6   | 6   | 6  || true
         -1  | -2  | 0  || false
         -28 | -26 | 26 || true
+        51  | 128 | 1  || true
```

Comment:
As you have added this weird test, your code now do not pass all tests correctly. Please remove your own test case.